vert.x/community/cache/booster.yaml

```diff
@@ -0,0 +1,22 @@
+metadata:
+  runsOn: 
+    - '!starter'
+    - '!osio'
+name: Vert.x Cache Booster
+description: Demonstrate how to use a cache server from an Eclipse Vert.x application
```

Comment:
Should mention that it's a "JDG cache server"?

Reply:
The name and description are not used anywhere, they can be safely removed

Reply:
no, JDG should not be mentioned.